core/templates/components/state-directives/answer-group-editor/answer-group-editor.directive.ts

```diff
@@ -78,14 +78,12 @@ angular.module('oppia').directive('answerGroupEditor', [
         'answer-group-editor.directive.html'),
       controllerAs: '$ctrl',
       controller: [
-        '$scope', '$rootScope', '$uibModal', 'StateInteractionIdService',
-        'AlertsService', 'ContextService', 'ExternalSaveService',
+        'StateInteractionIdService', 'AlertsService', 'ExternalSaveService',
```

Comment:
Can we sort these dependencies too? $ terms would come first, followed by the services, followed by the constants.